src/Stripe.net/Entities/StripeError.cs

```diff
@@ -4,28 +4,32 @@ namespace Stripe
 {
     public class StripeError : StripeEntity
     {
+        // For regular API errors:
+
         [JsonProperty("type")]
         public string ErrorType { get; set; }
 
+        [JsonProperty("charge")]
+        public string ChargeId { get; set; }
```

Comment:
Maybe put `ChargeId` above `ErrorType` and push `Message` down by a few parameters?

Reply:
Fixed!